apb-base/files/usr/bin/entrypoint.sh

```diff
@@ -34,20 +34,45 @@ if ! whoami &> /dev/null; then
 fi
 oc-login.sh
 
+
+set +x
+
+SECRETS_DIR=/etc/apb-secrets
+mounted_secrets=$(ls $SECRETS_DIR)
+
+extra_args=""
+if [[ ! -e "$mounted_secrets" ]] ; then
+
+    echo '---' > /tmp/secrets
```

Comment:
I wasn't totally sure about writing this to a file, but without some kind of transformation there isn't really a way to get the secrets into ansible, and I thought a file would be better than keeping it on the command line where it's likely to be echoed somewhere

Reply:
+1 makes sense